test/SlowTests/Voron/StreamBitArrayTests.cs

```diff
@@ -0,0 +1,66 @@
+﻿using FastTests;
+using Xunit;
+using Xunit.Abstractions;
+using Voron.Impl.FreeSpace;
+using Tests.Infrastructure;
+using Random = System.Random;
+
+namespace SlowTests.Voron
+{
+    public class StreamBitArrayTests : NoDisposalNeeded
+    {
+        public StreamBitArrayTests(ITestOutputHelper output) : base(output)
+        {
+        }
+   
+        [RavenFact(RavenTestCategory.Voron)]
+        public void VerifyResultWithRandomInput()
+        {
+            var random = new Random();
```

Comment:
Please use InlineDataWithRandomSeed so we'll pass and know the seed if it fails in the future

Reply:
done